locales/od_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -536,7 +538,8 @@ msgstr "ନାକଚ କରନ୍ତୁ"
 
 #. in https://duckduckgo.com/feedback under "Community Platform" header
 msgid "Care to help translate DuckDuckGo into your language?"
-msgstr "କଣ ଆପଣ ଡକଡକଗୋକୁ ନିଜ ଭାଷାରେ ଅନୁବାଦ କରିବାରେ ସାହାର୍ଯ୍ୟ କରିବାକୁ ଚାହୁଁଛନ୍ତି କି?"
+msgstr ""
+"କଣ ଆପଣ ଡକଡକଗୋକୁ ନିଜ ଭାଷାରେ ଅନୁବାଦ କରିବାରେ ସାହାର୍ଯ୍ୟ କରିବାକୁ ଚାହୁଁଛନ୍ତି କି?"
```

Comment:
Hi @soumendrak - sorry for the delay in responding.

I missed this in previous reviews, but since DuckDuckGo is a registered trade mark, the recommendation is usually to leave it untranslated.

Reply:
Hi @jbarrett - made the asked changes. Please check. Thanks.

Reply:
:+1: Many thanks!